cmd/lilypad/root.go

```diff
@@ -25,6 +25,9 @@ func NewRootCmd() *cobra.Command {
 	var network string
 	RootCmd.PersistentFlags().StringVarP(&network, "network", "n", "testnet", "Sets a target network configuration")
 
+	var powSignalCmd bool
+	RootCmd.PersistentFlags().BoolVarP(&powSignalCmd, "pow-signal", "", false, "Send a pow signal to smart contract")
```

Comment:
I'm a bit skeptical that the ideal solution is to add a persistent flag at the top level for cli params. Specially because this is now setting a `pow-signal` param to all commands (why would the `solver` and `job-creator` have to know anything about the `pow-signal` flag intended for a `resource-provider`?).

@noryev thoughts?